File: ios/Classes/DeepLinkReceivedEvent.swift
/* Copyright Airship and Contributors */

import Foundation

struct DeepLinkReceivedEvent: TiEvent {

    let name: String = EventNames.deepLinkReceived
    let data: [String: Any]
    init(deepLink: String) {
        self.data = ["deepLink": deepLink]
    }
}

File: android/src/ti/airship/events/DeepLinkReceivedEvent.kt
/* Copyright Airship and Contributors */
package ti.airship.events

import ti.airship.TiAirshipModule

class DeepLinkReceivedEvent(deepLink: String) : Event {
    override val data: Map<String, Any> = mapOf("deepLink" to deepLink)
    override val name: String = TiAirshipModule.eventDeepLinkReceived
}
